src/openvic-simulation/economy/ResourceGatheringOpertion.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "openvic-simulation/economy/ProductionType.hpp"
+#include "openvic-simulation/types/fixed_point/FixedPoint.hpp"
+#include "openvic-simulation/utility/Getters.hpp"
+
+#include "utility/Constructors.hpp"
+
+namespace OpenVic {
+	class ResourceGatheringOpertion final {
+	public:
+		typedef ordered_map<Pop* const, Pop::pop_size_t> employees_map_t;
+		ProductionType const& PROPERTY(production_type);
+		fixed_point_t PROPERTY(revenue_yesterday);
+		fixed_point_t PROPERTY(output_quantity_yesterday);
+		fixed_point_t PROPERTY(unsold_quantity_yesterday);
+		fixed_point_t PROPERTY(size_multiplier);
+		ordered_map<Pop* const, Pop::pop_size_t> PROPERTY(employees);
```

Comment:
`employees` isn't using the `employees_map_t` alias here, which should also be defined with `using` rather than `typedef`. Also the `Pop` pointer should be non-const here too.